src/basic/verbs.c

```diff
@@ -24,6 +24,7 @@
 #include <stddef.h>
```

Comment:
the commit msg is incorrect, it refers to argv[0]? but it's argv[optind] we talk about here, which is initially argv[1], no?

Reply:
hmm, can you elaborate on the difference between VERB_DEFAULT and VERB_FALLBACK?